CPAC/pipeline/cpac_pipeline.py

```diff
@@ -769,6 +783,17 @@ def run_workflow(sub_dict, c, run, pipeline_timing_info=None, p_name=None,
                                  c['subject_id'])
                 ))
 
+                if workflow_save:
+                    workflow_filename = os.path.join(
+                        log_dir,
+                        f'workflow_post_{workflow_meta.time.strftime("%Y-%m-%d_%H-%M-%S")}_{p_name}.json'
```

Comment:
```suggestion
                        'workflow_post_'
                        f'{workflow_meta.time.strftime("%Y-%m-%d_%H-%M-%S")}_{p_name}.json'
```
for line length

Reply:
Sorry I forgot to push a commit that adds a helper function here.

Reply:
170893e687e7df304c7ea7259c8f8d59c99f2049